### ISSUE TITLE
Procedure for booting a non-sharded replica set?

### ISSUE THREAD

**Issue by markbate** [edelight/chef-mongodb#35]
Hi,
I'm having a lot of fun trying to setup a replica set. 
I've followed the docs (call the mongodb::replicaset recipe & set node[:mongodb][:cluster_name), but it always fail with "cannot configure replicaset 'rs_wunderkit', no member nodes found".

I found issues #14 & #6 and have followed the advice of setting the "shard_name", but things still fail. I also found the wiki entry, but it didn't seem to make sense in this case as I'm not sharding.

I thought that perhaps I first need to boot the nodes as standalone nodes with the cluster_name & shard_name set, then do a second run with the replicaset recipe, but that didn't work either.

can you please describe how one should go about setting up that first replica node?
thanks

**Comment by markbate** [comment]
Er, that might have been my bad on that one.
I was using librarian-chef & calling the mongodb cookbook from the community repo:

site 'http://community.opscode.com/api/v1'
cookbook 'mongodb'

I noticed the cookbook was recently moved to a top level, so I tried going directly to the git repo:

cookbook 'mongodb',
  :git => 'https://github.com/edelight/chef-mongodb'

Things now seem to work as described in #14 - the run will warn, but not fail.

Interestingly though, the version numbers of the cookbooks are the same, so it was only through running that I found the fix.

Thanks, sorry for the mix up.
